qsignature/src/org/qcmg/sig/messages.properties

```diff
@@ -1,6 +1,8 @@
-USAGE = usage: qsignature [org.qcmg.sig.SignatureGeneratorBespoke | org.qcmg.sig.CompareGenotypeBespoke] [-options]
-COMPARE_USAGE = usage: qsignature org.qcmg.sig.Compare -log <log_file> -dir <directory in which to search for .qsig.xml files> -output <output xml file>
-GENERATOR_USAGE = usage: qsignature org.qcmg.sig.SignatureGeneratorBespoke -snpPositions <file containing positions of interest> -illuminaArraysDesign <Illumina arrays design document - contains list of snp ids and whether they should be complemented> -input <bam or snp chip file, or folder containing bams or snp chip files> -output <output directory where qsig.xml files will be written to> -log <log_file>
+USAGE = usage: qsignature.jar [org.qcmg.sig.SignatureGeneratorBespoke | org.qcmg.sig.Compare | org.qcmg.sig.VcfProfiler] [-options]
```

Comment:
Should the usage start with  "USAGE = usage: java -cp qsignature.jar ..." ?

Reply:
Looks like some duplication of `usage` here.
Does JP's document have any guidance here?

Reply:
All our tools have this (well, all the ones I have checked). If we want to change this, it should be to all tools and in a seperate PR.

Reply:
In JP's modified document, all usage includes "java -jar <tool>.jar ..." or "java -cp <tool>.jar ...".  We used to have a batch script to run the tool, but not now.  However we never have "usage: <tool>.jar ..."
It is better to make it consistent with our document. There are many tools already followed JP's suggestion, please refer to q3tiledaligner, qprofiler, q3vcftools, qbamfilter, qmito.

Reply:
> In JP's modified document, all usage uses "java -jar tool.jar ..." or "java -cp tool.jar ...". We used to have a batch script to run the tool, but not now. However we never have "usage: tool.jar ..."
> It is better to make it consistent with our document. There are many tools already followed JP's suggestion, please refer to q3tiledaligner, qprofiler, q3vcftools, qbamfilter, qmito.

Ah - apologies - I thought you were referring to "usage" in the usage.
Updated